cg/cli/workflow/commands.py

```diff
@@ -89,6 +91,29 @@ def link(context: CGConfig, case_id: str, dry_run: bool):
     analysis_api.link_fastq_files(case_id=case_id)
 
 
+@click.command("metrics-deliver")
+@ARGUMENT_CASE_ID
+@OPTION_DRY
+@click.pass_obj
+def metrics_deliver(context: CGConfig, case_id: str, dry_run: bool) -> None:
+    """Create and validate a metrics deliverables file for given case id.
+    If QC metrics are met it sets the status in Trailblazer to complete.
+    If failed, it sets it as failed and adds a comment with information of the failed metrics."""
+
+    analysis_api: RnafusionAnalysisAPI | TaxprofilerAnalysisAPI = context.meta_apis["analysis_api"]
+
+    try:
+        analysis_api.status_db.verify_case_exists(case_internal_id=case_id)
+    except CgError as error:
+        raise click.Abort() from error
```

Comment:
We need to make sure that this click.Abort() doesn't disrupt the general store-availble function. Someone mentioned recently that this could be problematic (in the context of rnafusion), but I haven't checked yet (so still unclear). 

Nothing to do here, just a note for you to know

Reply:
I will resolve this for now.